ray-operator/config/samples/ray_v1alpha1_rayservice.yaml

```diff
@@ -3,4 +3,94 @@ kind: RayService
 metadata:
   name: rayservice-sample
 spec:
-  # TODO(user): Add fields here
+  serveConfigs:
+    - name: shallow
+      import_path: "test_env.shallow_import.ShallowClass"
+      num_replicas: 1
+      route_prefix: "/shallow"
+      ray_actor_options:
+        num_cpus: 0.1
+        runtime_env:
+          py_modules:
+            - "https://github.com/shrekris-anyscale/test_deploy_group/archive/HEAD.zip"
```

Comment:
@shrekris-anyscale can you please transfer the repo to ray-project?

Reply:
or make a copy

Reply:
what about `test_module` ?

Reply:
Ah, I hadn't ported that one yet. Can you try this link:

https://github.com/ray-project/test_module/archive/aa6f366f7daa78c98408c27d917a983caa9f888b.zip

Reply:
@shrekris-anyscale could you help to improve the runtime_env config to make them general enough that we can merge it in the kuberay repo? Thanks!

Reply:
@Jeffwan We're using two zip files in this case for testing purposes. They're essentially two different codebases. See [this documentation](https://docs.ray.io/en/releases-1.12.1/ray-core/handling-dependencies.html#remote-uris) for how these zip files are generated. You can also use S3 and GCS, but GitHub is convenient when writing tests because people can pull or push new code and update the URL more easily.

Reply:
I synced with @brucez-anyscale offline. The runtime_env can use S3 and Google Storage URLs, but GitHub is more convenient for testing.